knowledge-base/actions/uraimo/run-on-arch-action/action-security.yml

```diff
@@ -0,0 +1,2 @@
+name: 'Run on architecture'
+#No Permissions Needed
```

Comment:
I think permission here should be `packages: write`, `to cache docker image in package registry`
view [here](https://github.com/uraimo/run-on-arch-action/blob/b818ac073677cb26a21e1dd897cbfe5a3726adf0/action.yml#L16) for adding action input.